lib/ansible/modules/network/f5/wait_for_bigip.py

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: wait_for_bigip
```

Comment:
I know it may not read as well, but I hough could we call this `bigip_wait_for` to be in line with the other modules?

May make running the tests easier, as we use the platform (bigip, iosxr, vyos) as the prefix.

Reply:
@gundalow which tests? the unit ones?

Reply:
@gundalow would `bigip_wait` be acceptable?

Reply:
When we hook Distributed CI (DCI) for F5 we invoke tests by doing
`/ansible-test network-integration {{ platform }}_.* -v`

Where `platform` is eos, ios, iosxr, bigip, vyos, etc

`bigip_wait` sounds good